### PR TITLE
Add week 1 assignment

### PR DESCRIPTION
**Pull request by 0x0kurooo**
Grabvn Golang bootcamp week 1 assignment

### CODE REVIEW

**Comment by PrimeTimeTran** [comment]
Looks good to me =)

File: week-1/lib.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+import (
+	"errors"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+const (
+	PARAMS_NUM = 3
```
Comment (hoksilato) [inline week-1/lib.go]: Golang has [`iota`](https://github.com/golang/go/wiki/Iota) identifier and you can use it for this. And you can use lowercase for constants in Go, which one starts with capital will be Exported constants (which can be used in other packages).

File: week-1/lib.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+	case "(":
+		return -1
+	case ")":
+		return -2
+	default:
+		return 0
+	}
+}
+
+func isOperator(op string) bool {
```
Comment (hoksilato) [inline week-1/lib.go:25]: Better name: `isValidOperator`.

File: week-1/lib.go [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+	switch op {
+	case "+", "-", "*", "/":
+		return true
+	default:
+		return false
+	}
+}
+
+func normalization(exp string) string {
```
Comment (hoksilato) [inline week-1/lib.go:34]: You can use [`strings.Fields`](https://golang.org/pkg/strings/#Fields), it can replace this function.

**Review by hoksilato — COMMENTED** [review]
Excellent work 💯 , thank you so much for your extra work. Hope you had fun with Go.

**Comment by 0x0kurooo** [comment]
> Excellent work  , thank you so much for your extra work. Hope you had fun with Go.

Thank for your advises. It very helpful.

**Pull request merged**